python/cugraph/cugraph/structure/property_graph.py

```diff
@@ -1174,9 +1182,9 @@ def select_edges(self, expr):
         >>> pG = PropertyGraph()
         >>> pG.add_edge_data(df, type_name="etype", vertex_col_names=("src", "dst"))
         >>> vert_df = cudf.DataFrame({"vert_id": [99, 22, 98, 34, 97, 56, 96, 88],
-        ...                           "v_prop": [1 ,2 ,3, 4, 5, 6, 7, 8]})
-        >>> pG.add_vertex_data(vert_df, type_name="etype", vertex_col_name="vert_id")
-        >>> selection = pG.select_edges("(_TYPE_ == 'etype') &
+        ...                           "v_prop": [1, 2, 3, 4, 5, 6, 7, 8]})
+        >>> pG.add_vertex_data(vert_df, type_name="vtype", vertex_col_name="vert_id")
+        >>> selection = pG.select_edges("(_TYPE_ == 'vtype') &
```

Comment:
This example doesn't work for me as-is.  The call to `pG.select_edges` is being passed a query for 'vtype', but that's not a type that was added to edges in the `add_edge_data` call above.
```suggestion
        >>> selection = pG.select_edges("(_TYPE_ == 'etype') &
```